src/sys/socket/mod.rs

```diff
@@ -182,6 +188,11 @@ pub enum ControlMessage<'a> {
     /// "Ancillary messages" section of the
     /// [unix(7) man page](http://man7.org/linux/man-pages/man7/unix.7.html).
     ScmRights(&'a [RawFd]),
+    /// A message of type SCM_TIMESTAMP, containing the time the packet
```

Comment:
This comment is too long so it looks terrible in the generated docs. You should generate the docs on your end to check (`cargo doc --no-deps --open`) and search for ScmTimestamp and you'll see what I'm talking about.

All docs should have a short descriptive sentence at the start, followed by an empty comment line, then a more descriptive text. Please see [the Rust book](https://doc.rust-lang.org/stable/book/first-edition/documentation.html) for some more details on writing effective documentation.